src/service/utils.py

```diff
@@ -74,3 +74,18 @@ def remove_tool_calls(content: str | list[str | dict]) -> str | list[str | dict]
         for content_item in content
         if isinstance(content_item, str) or content_item["type"] != "tool_use"
     ]
+
+
+def interrupt_from_call_tool(call_tool: dict) -> InterruptMessage:
+    # Crear instancia de InterruptMessage a partir del diccionario call_tool
```

Comment:
Would you mind putting code comments in english to be consistent with the rest of the repo?